src/GitVersionCore.Tests/Core/GitVersionExecutorTests.cs

```diff
@@ -418,7 +418,7 @@ public void GetProjectRootDirectoryWorkingDirectoryWithWorktree()
 
                 sp = GetServiceProvider(gitVersionOptions);
 
-                gitVersionOptions.ProjectRootDirectory.TrimEnd('/', '\\').ShouldBe(worktreePath);
```

Comment:
Note: `ProjectRootDirectory` was actually being set to the "working directory" of the git repository. I renamed it to `GitRepositoryWorkingDirectory` to aid understanding whilst refactoring. It's not a critical change.

Reply:
This has been a constant source of friction, because I think it's used to support dynamic repositories in a weird way I don't quite understand. I think it's been changed back and forth several times in different PR's, each breaking one other use case that isn't well enough tested. 🤷‍♂️